corehq/apps/app_manager/models.py

```diff
@@ -5707,63 +5707,89 @@ def reapply_overrides(self):
 
 
 def import_app(app_id_or_source, domain, source_properties=None, request=None, check_all_reports=True):
+    source_app = _get_source_app(app_id_or_source)
+    source = source_app.export_json(dump_json=False)
+
+    attachments = _get_attachments(source)
+    source['_attachments'] = {}
```

Comment:
Any idea why we set `_attachments` to an empty dict after reading from it? This feels like it could use a comment.

Reply:
I see that have to be moved to the new app, but I don't know why they're cleared from the old app.